sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/datastore/V1Beta3ReadIT.java

```diff
@@ -0,0 +1,91 @@
+package org.apache.beam.sdk.io.gcp.datastore;
+
+import static org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.deleteAllEntities;
+import static org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.getDatastore;
+import static org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.makeAncestorKey;
+import static org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.makeEntity;
+
+import org.apache.beam.sdk.Pipeline;
+import org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.UpsertMutationBuilder;
+import org.apache.beam.sdk.io.gcp.datastore.V1Beta3TestUtil.V1Beta3TestWriter;
+import org.apache.beam.sdk.options.PipelineOptionsFactory;
+import org.apache.beam.sdk.testing.PAssert;
+import org.apache.beam.sdk.testing.TestPipeline;
+import org.apache.beam.sdk.transforms.Count;
+import org.apache.beam.sdk.values.PCollection;
+
+import com.google.datastore.v1beta3.Entity;
+import com.google.datastore.v1beta3.Key;
+import com.google.datastore.v1beta3.Query;
+import com.google.datastore.v1beta3.client.Datastore;
+
+import org.junit.After;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.JUnit4;
+
+import java.util.UUID;
+
+/**
+ * End-to-end tests for Datastore V1Beta3.Read.
+ */
+@RunWith(JUnit4.class)
+public class V1Beta3ReadIT {
+  private V1Beta3TestOptions options;
+  private String ancestor;
+  private final long numEntities = 1000;
+
+  @Before
+  public void setup() {
+    PipelineOptionsFactory.register(V1Beta3TestOptions.class);
```

Comment:
I tried removing it but ran into some this Exception: ""IllegalArgumentException: missing a property namespace ... " from [here](https://github.com/apache/incubator-beam/blob/81dcd527888be7bcfde09e47a3f512aeafd4a962/sdks/java/core/src/main/java/org/apache/beam/sdk/options/PipelineOptionsFactory.java#L1397). I will work with Jason to debug this in more detail and remove this line if needed if a different PR.